src/app/compiler/compiler-imports.js

```diff
@@ -14,16 +15,16 @@ const profile = {
 }
 
 module.exports = class CompilerImports extends Plugin {
-  constructor (githubAccessToken) {
+  constructor () {
     super(profile)
-    this.githubAccessToken = githubAccessToken || (() => {})
     this.previouslyHandled = {} // cache import so we don't make the request at each compilation.
   }
 
   handleGithubCall (root, path, cb) {
     let param = '?'
-
-    param += this.githubAccessToken() ? 'access_token=' + this.githubAccessToken() : ''
+    // const token = await this.call('settings', 'getGithubAccessToken')
+    const token = globalRegistry.get('config').api.get('settings/gist-access-token') // TODO replace with the plugin call above
+    param += token ? 'access_token=' + token : ''
```

Comment:
Maybe more readable to do : 
```typescript
const param = token ? `?access_token=${token}` : '?' 
```
(or "let param = ..." if you're mutating the param later.